servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpApiConversions.java

```diff
@@ -353,17 +378,31 @@ public static ServiceAdapterHolder toStreamingHttpService(BlockingHttpService se
     }
 
     /**
-     * Convert from a {@link BlockingStreamingHttpService} to a {@link StreamingHttpService}.
+     * Convert from a {@link BlockingStreamingHttpService} to a {@link ServiceAdapterHolder}.
```

Comment:
BlockingStreamingHttpService -> BlockingHttpService, same in lines 365, 367, 383, 395, 398